javascript/ql/test/library-tests/TaintTracking/exceptions.js

```diff
@@ -58,7 +58,7 @@ function test(unsafe, safe) {
     try {
       await throwAsync(source());
     } catch (e) {
-      sink(e); // NOT OK
+      sink(e); // NOT OK - but not flagged
```

Comment:
Why do we miss this now? Are we missing a load step from the `await` operand to the catch?

Reply:
I think it is because of the mix of calls and property store/read. 

The flow is: flow into `throwAsync()` -> store in pseudo-property -> return from `throwAsync()` -> read pseudo-property -> local flow to `e`.

Reply:
And the new code in `loadStep`/`storeStep` doesn't catch it, because they only handle ordinary returns and not exceptional returns.